hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeGraphAuthProxy.java

```diff
@@ -1130,6 +1158,7 @@ public HugeUser deleteUser(Id id) {
             E.checkArgument(!HugeAuthenticator.USER_ADMIN.equals(user.name()),
                             "Can't delete user '%s'", user.name());
             verifyUserPermission(HugePermission.DELETE, user);
+            auditPerUserRates.invalidate(IdGenerator.of(user.name()));
```

Comment:
this.auditPerUserRates

Reply:
this method is in `AuthManagerProxy` (inner class), cannot use `this`

so as `updateCreator`